src/components/Requirements.vue

```diff
@@ -329,7 +349,7 @@ export default Vue.extend({
           .then(result => {
             result.data.courses.forEach((course: FirestoreSemesterCourse) => {
               // @ts-ignore [We should resolve this later]
-              const createdCourse = this.$parent.createCourse(course, true);
+              const createdCourse = this.$parent.createCourse(course, true, false);
```

Comment:
Since it's only one $parent and we're at it can we swap out this DOM manip with $emit instead?

Reply:
@hanywang2 This is a Monday item that I was trying to work on in a previous PR but I didn't know how to get the created course using $emit. Would we pass the created course as a prop to the child components?

Reply:
@hanywang2 The ultimate goal is probably not to call `createCourse()` for requirements bar courses at all because it seems to cause issues when trying to set up the firestore user data listener (See @SamChou19815 's note in #217 ). I will try to address this in my future PR for resolving dragging issues.